objectivec/Tests/GPBPerfTests.m

```diff
@@ -64,6 +64,57 @@ - (void)testMessagePerformance {
   }];
 }
 
+- (void)testMessageSerialParsingPerformance {
```

Comment:
Actually, can you tweak the name/add a comment to say what this (and the next one) are for?  Call out that they are specific to *parsing* (the other is building, serializing, and parsing), and that it is for extension fields.

It might not hurt to also make a version that uses TestAllTypes that just covers fields and not extensions.